src/store/store.js

```diff
@@ -12,9 +12,7 @@ const configureStore = () => {
   const store = createStore(
     rootReducer(history),
     compose(
-      applyMiddleware(routerMiddleware(history), sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
Please, revert this change, we still need to have redux devtools for the development